### PR TITLE
Migrated to Github actions

### LINKED ISSUE #8: Migrate to Github Actions

### PR DESCRIPTION
**Pull request by RishiKumarRay**
Fixed #8

### CODE REVIEW

**Comment by RishiKumarRay** [comment]
I have added a screenshot too 
Screenshot from 2022-01-24 21-08-37

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches: [ main ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2.3.1
+        with:
+         python-version: "3.6"
```
Comment (Daltz333) [inline .github/workflows/ci.yml]: Can we use Python 3.8?

Comment (RishiKumarRay) [reply .github/workflows/ci.yml]: Yeah we can , actually the azure pipeline had 3.6 version thats why i took this

**Comment by RishiKumarRay** [comment]
@Daltz333 i have updated the version please do review now

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**